x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -256,11 +256,6 @@ public enum Cap {
          */
         COMBINE_BINARY_COMPARISONS,
 
-        /**
-         * MATCH command support
-         */
-        MATCH_COMMAND(true),
-
```

Comment:
I think that it is safe to remove the capability. @thecoop  ??

Reply:
Yeah, capabilities are dynamic and can be removed if they no longer apply